components/usage/pkg/stripe/stripe.go

```diff
@@ -72,3 +109,8 @@ func queriesForCustomersWithTeamIds(teamIds []string) []string {
 
 	return queries
 }
+
+// workspaceSecondsToCredits converts seconds (of workspace usage) into Stripe credits.
+func workspaceSecondsToCredits(seconds int64) int64 {
+	return (seconds + 59) / 60
```

Comment:
Oh, that's an interesting way to round up. I guess this is equivalent to something like `Math.ceil(seconds / 60)`?

Reply:
yes, this is an easy way to ensure that we bill partial minutes as full minutes.